src/analytics/trackingEvents.ts

```diff
@@ -0,0 +1,27 @@
+// list of tracking events
+export enum GenericEvents {
+  pressedButton = 'Pressed Button',
+  pressedButton2 = 'Pressed Button 2',
+}
+
+export enum SwapEvents {
+  submittedSwap = 'Submitted Swap',
```

Comment:
Some events are string-concatenated at the call site 🤔 I wonder what we can do for those...

Reply:
i'm under the belief that string concatenated events are terrible, same effect can be down with a standardized event name and adding a property to the event